infra/ansible/roles/dns_ospf/templates/iptables.j2

```diff
@@ -10,6 +10,7 @@
 {% if INTERNAL_LISTEN_IP != "" %}
 -A INPUT -d {{ INTERNAL_LISTEN_IP }}/32 -p tcp -m tcp --dport 53 -j ACCEPT
 -A INPUT -d {{ INTERNAL_LISTEN_IP }}/32 -p udp -m udp --dport 53 -j ACCEPT
+-A INPUT -p icmp --icmp-type echo-request -j ACCEPT
```

Comment:
Should we limit it to only the internal IP?

Reply:
Good call